flock_internal_test.go

```diff
@@ -0,0 +1,30 @@
+package flock
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func Test(t *testing.T) {
+	tmpFileFh, err := ioutil.TempFile(os.TempDir(), "go-flock-")
+	tmpFileFh.Close()
+	tmpFile := tmpFileFh.Name()
+	os.Remove(tmpFile)
+
+	lock := New(tmpFile)
+	locked, err := lock.TryLock()
+	if locked == false || err != nil {
+		t.Fatal("failed to lock")
```

Comment:
If this were to fail it might be helpful to know which it was. How would you feel about `t.Fatalf("failed to lock: locked: %t, err: %v", locked, err)`?

Reply:
```suggestion
		t.Fatalf("failed to lock: locked: %t, err: %v", locked, err)
```